src/py/flwr/server/superlink/linkstate/in_memory_linkstate.py

```diff
@@ -448,6 +448,20 @@ def get_run(self, run_id: int) -> Optional[Run]:
                 return None
             return self.run_ids[run_id].run
 
+    def get_run_timestamps(self, run_id: int) -> tuple[str, str, str, str]:
+        """Retrieve the timestamps for the specified `run_id`."""
+        with self.lock:
+            if run_id not in self.run_ids:
+                log(ERROR, "`run_id` is invalid")
+                return "", "", "", ""
+            run_record = self.run_ids[run_id]
+            return (
```

Comment:
@panh99 I'm wondering if we could return this as a dict instead, so we are a step closer to the JSON-compatible format @chongshenng mentioned yesterday. (The same would apply to the SQL Linkstate)